boa/src/builtins/mod.rs

```diff
@@ -6,6 +6,7 @@ pub mod boolean;
 pub mod console;
 pub mod error;
 pub mod function;
+pub mod globalThis;
```

Comment:
The rust language use `snake_case` for module names, same for variables, and `RascalCase` for type definitions. for more information on this you can see: <https://rust-lang.github.io/api-guidelines/naming.html>
The language is very strict about naming conventions so it does not compile the code.

So this we need to make this snake case so this would be `global_this`, instead of `globalThis`
Note the file name will have to change too.

and in the `init` function in `builtins/mod.rs` we call `global_this::init()`, instead of `globalThis::init()`

Reply:
... I'm embarrassed.. I knew this, but I think I got too excited. My apologies, I'm fixing it now.